src/routes/withdraw/withdraw-form.tsx

```diff
@@ -69,13 +69,19 @@ export const WithdrawForm = ({
     }
   }, [status, history]);
 
+  React.useEffect(() => {
+    const run = async () => {
+      await setAddressValid(ethers.utils.isAddress(destinationAddress));
```

Comment:
You're awaiting on nothing here because the state setter doesn't return anything.

Reply:
Also ethers.utils.isAddress doesn't return a promise at all so theres no need to use an async function here